<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TranslatorWidget</class>
 <widget class="QWidget" name="TranslatorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>144</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="engineComboBox">
       <property name="toolTip">
        <string>Select the translation service</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="preferencesButton">
       <property name="toolTip">
        <string>Press to open the Translator configuration page</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QToolButton" name="pronounceOrigButton">
       <property name="toolTip">
        <string>Press to pronounce the entered text</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="origLanguageComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Select the language of the original text</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="swapButton">
       <property name="toolTip">
        <string>Press to swap the translation direction</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="transLanguageComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Select the language for the translated text</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QToolButton" name="pronounceTransButton">
       <property name="toolTip">
        <string>Press to pronounce the translated text</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" rowspan="4" colspan="2">
      <widget class="QPlainTextEdit" name="origEdit">
       <property name="toolTip">
        <string>Enter the text to be translated</string>
       </property>
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>23</width>
         <height>68</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="3" rowspan="4" colspan="2">
      <widget class="QTextEdit" name="transEdit">
       <property name="toolTip">
        <string>Shows the translated text</string>
       </property>
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QToolButton" name="translateButton">
       <property name="toolTip">
        <string>Press to translate the entered text</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QToolButton" name="clearButton">
       <property name="toolTip">
        <string>Press to clear the text fields</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>23</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>engineComboBox</tabstop>
  <tabstop>pronounceOrigButton</tabstop>
  <tabstop>origLanguageComboBox</tabstop>
  <tabstop>swapButton</tabstop>
  <tabstop>transLanguageComboBox</tabstop>
  <tabstop>pronounceTransButton</tabstop>
  <tabstop>origEdit</tabstop>
  <tabstop>translateButton</tabstop>
  <tabstop>transEdit</tabstop>
  <tabstop>clearButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
